Upstream-Status: Inappropriate [board-specific]

diff --git a/arch/arm/boot/dts/ti/omap/am335x-boneblack.dts b/arch/arm/boot/dts/ti/omap/am335x-boneblack.dts
index 16b567e3cb47..9767bddcdd2d 100644
--- a/arch/arm/boot/dts/ti/omap/am335x-boneblack.dts
+++ b/arch/arm/boot/dts/ti/omap/am335x-boneblack.dts
@@ -173,3 +173,22 @@ &gpio3 {
 &baseboard_eeprom {
 	vcc-supply = <&ldo4_reg>;
 };
+
+&am33xx_pinmux {
+      dcan0_pins: dcan0-pins {
+          pinctrl-single,pins = <
+              AM33XX_PADCONF(AM335X_PIN_UART1_CTSN, PIN_OUTPUT_PULLUP, MUX_MODE2)
+              AM33XX_PADCONF(AM335X_PIN_UART1_RTSN, PIN_INPUT_PULLUP, MUX_MODE2)
+          >;
+      };
+};
+
+&dcan0 {
+      pinctrl-names = "default";
+      pinctrl-0 = <&dcan0_pins>;
+      status = "okay";
+};
+
+&i2c2 {
+      status = "disabled";
+};
